config/sidekiq_production.yml

```diff
@@ -0,0 +1,18 @@
+SparklineScheduler:               # rake cache_refresh:sparkline
+  cron: '0 0 19 * * * Etc/UTC'
+
+CoinSlugsScheduler:               # rake generate_coin_slugs
+  cron: '0 0 1 * * * Etc/UTC'
+
+SitemapScheduler:                 # rake sitemap:refresh
+  cron: '0 0 0 * * * Etc/UTC'
+
+staked_cofi_perform_actions:      # rake staked_cofi:perform_actions
+  cron: '0 30 * * * *'
```

Comment:
Thanks for including the timezone in the previous cron declarations; let's stick with that here as well for consistency.